cmd/dep/prune.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+
+	"github.com/golang/dep"
+)
+
+const pruneShortHelp = `Prune was merged into ensure. Use ensure instead.`
+const pruneLongHelp = `
+Prune was merged into the ensure command.
+Set prune options in the manifest and it will be applied after every ensure.
+`
```

Comment:
Add to the last line:

dep prune will be removed in a future version of dep. Please update your scripts.